test/unit-test/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# Include filepaths for source and include.
+include( ${MODULE_ROOT_DIR}/source/dependency/coreMQTT/mqttFilePaths.cmake )
+include( ${MODULE_ROOT_DIR}/mqttAgentFilePaths.cmake )
```

Comment:
I don't think these are includes are necessary anymore since they are included in `test/CMakeLists.txt`